app.py

```diff
@@ -34,16 +35,16 @@ def init_app(self):
                                  "a current Shot, current Asset etc). This app requires "
                                  "an entity as part of the context in order to work.")
 
-        self.engine.register_command("Sync Frame Range with Shotgun", self.run_app)
+        self.engine.register_command("Sync Frame Range with Shotgun",
+                                     lambda: self.run_app(from_menu=True))
```

Comment:
Excuse me for this, but what is the lambda doing here? Not sure to full understand.